userspace/libsinsp/CMakeLists.txt

```diff
@@ -171,12 +171,19 @@ endif()
 
 target_include_directories(sinsp
 PUBLIC
-	${CMAKE_CURRENT_SOURCE_DIR}
-	${CMAKE_CURRENT_SOURCE_DIR}/../plugin
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../plugin>
+	$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/${LIBS_PACKAGE_NAME}>
+	$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/${LIBS_PACKAGE_NAME}/libsinsp>
```

Comment:
@therealbobo Could you please check this? After your recent changes does it still make sense to have an include path that enters inside the `libsinsp/` directory? I suppose it does if you want both these includes to work:
```c++
#include <libsinsp/sinsp.h>
#include <sinsp.h>
```
It would be nice to have a single approach, though.

Reply:
I have to do some more cleanups of my leftovers! But yes, I completely agree on that! Let's go with `<libsinsp/*>` 😄

Reply:
@therealbobo This PR has been stalling for a while, I have just rebased it to fix a conflict.
Are you expecting me to remove the support for the headers with no path (e.g. `#include <sinsp.h>`) in this PR? I can do that, but I need clear indication that it is ok and expected.

Reply:
Yes, exactly! I did a deep cleanup recently. I don't think that there's any header with a relative path.